### PR TITLE
[t1069] fixed page-element hilight/blink

### PR DESCRIPTION
**Pull request by secretrobotron**
1. Safety check for tracks already on media in timeline/media.js
2. Used ontransitionend instead of timeout to capture blink ending

### CODE REVIEW

File: src/ui/page-element.js [hunk 1]
```diff
@@ -53,17 +53,24 @@ define( [ "core/logger", "core/eventmanager", "util/dragndrop", "ui/position-tra
...
-      setTimeout(function(){
-        if( _highlightElement.getAttribute( "data-blink" ) !== true ){
-          _highlightElement.removeAttribute( "data-blink" );
-          _highlightElement.style.visibility = "hidden";
-        }
-        _this.blink = _blinkFunction;
-      }, 1500 );
+      _highlightElement.addEventListener( 'transitionend', onTransitionEnd, false );
+      _highlightElement.addEventListener( 'oTransitionEnd', onTransitionEnd, false );
+      _highlightElement.addEventListener( 'webkitTransitionEnd', onTransitionEnd, false );
```
Comment (jbuck) [inline src/ui/page-element.js:73]: no mozTransitionEnd? :(

Comment (jbuck) [inline src/ui/page-element.js:73]: Oh wait, Firefox supports standard transitionend. My bad!

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:80/9003f8446a9bbb6/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:80/55dfdd7b237a6df/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:80/9003f8446a9bbb6/output.txt

Total script time: 0.21 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:80/55dfdd7b237a6df/output.txt

Total script time: 0.49 mins

Server is running on http://pr314.butyr.org/templates/test.html

**Pull request merged**